### ISSUE TITLE
Save window groups

### ISSUE THREAD

**Issue by jrdnbwmn** [zeit/hyperterm#311]
Idea for a new feature: ability to save window groups [like you can in Terminal](http://osxdaily.com/2013/12/09/use-manage-terminal-window-groups-mac/). 

This is something I and a lot of other devs I know use all the time. I looked through current functionality and plugins and couldn't find anything that does this.

May be best as a plugin.

**Comment by felipemrodrigues** [comment]
+1

**Comment by rdewolff** [comment]
+1

**Comment by chabou** [comment]
Saving windows/tabs/panes and their working directory is enough?
Or restoring remote SSH sessions is a big part of this ?

**Comment by cbrwizard** [comment]
https://github.com/timolins/hyperlayout seems to solve this

**Comment by jaredramirez** [comment]
+1

**Comment by felipemrodrigues** [comment]
@cbrwizard nice tip

**Comment by cguodesign** [comment]
+1

**Comment by gurmukhp** [comment]
Any update on this? I'd love to be able to remember window layouts on reload.

**Comment by alongenosar** [comment]
+1

**Comment by ppot** [comment]
I could make this as a plugin ;) before session restoration if it's only the windowa

**Comment by alongenosar** [comment]
you mean panes?

**Comment by ppot** [comment]
@alongenosar I could hack on this this weekend. Well see what I cam up with. If you only want one save at the end I can invoke something. But if you want memory session restoration, i'm not there yet. Backend dosen't support this as a plugin.

**Comment by alongenosar** [comment]
great

**Comment by ppot** [comment]
What i'm working on is a way to: Set an environment.
What I meant is
1. Open window and pane etc.
2. Save as profile.
3. Define base profile on startup.
4. Open different profile.

**Comment by alongenosar** [comment]
Sounds great! 
let me know when I can test it.

**Comment by ppot** [comment]
Target 1: update Hyper window object: #3485

**Comment by NeonWizard** [comment]
Hyperlayout seems to be discontinued and is no longer accepting pull requests. I would very much prefer if this basic functionality was provided in vanilla Hyper, it's been asked many times over the last several years.

**Comment by linareyne** [comment]
Yeah, still nothing it looks like? This is such a basic feature, it's criminal for hyper to not have it. Makes it not very suited to use for a work environment, unfortunately.
